.ci-config/project.properties

```diff
@@ -0,0 +1 @@
+guava|git|https://github.com/google/guava|v28.2||
```

Comment:
We do not need outside project.
We should avoid network downloads.

You have java 2 projects(in one git repository) at this repo - use them.
`extractor|git|files://.|main`

Uri to folder of repo should work, git works fine with `file://` protocol.
We even used it somewhere in past.

Reply:
done.